src/NServiceBus.Metrics.ServiceControl/NServiceBus.Metrics.ServiceControl.csproj

```diff
@@ -2,18 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
-    <OutputPath>..\..\binaries\</OutputPath>
-    <DocumentationFile>..\..\binaries\net452\NServiceBus.Metrics.ServiceControl.xml</DocumentationFile>
-    <LangVersion>7</LangVersion>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <DebugType>full</DebugType>
-    <DebugSymbols>true</DebugSymbols>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
+    <Description>Plugin for sending endpoint monitoring data to a ServiceControl Monitoring instance</Description>
   </PropertyGroup>
+
   <ItemGroup>
-    <PackageReference Include="GitVersionTask" Version="3.6.5" />
-    <PackageReference Include="NServiceBus" Version="[6.4.0,7.0.0)" />
-    <PackageReference Include="NServiceBus.Metrics" Version="[2.0.0-*,3.0.0)" />
-    <PackageReference Include="NuGetPackager" Version="0.6.5" />
-    <PackageReference Include="ServiceControl.Monitoring.Data" Version="2.1.1" />
+    <PackageReference Include="NServiceBus" Version="[6.4.0, 7.0.0)" />
```

Comment:
The other two PRs produce packages that have the latest patch version as the minimum. Should this be `[6.4.4, 7.0.0)` instead?